dynawo/sources/Models/Modelica/Dynawo/Electrical/Loads/LoadAlphaBetaRestorative.mo

```diff
@@ -0,0 +1,48 @@
+within Dynawo.Electrical.Loads;
+
+/*
+* Copyright (c) 2015-2020, RTE (http://www.rte-france.com)
+* See AUTHORS.txt
+* All rights reserved.
+* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, you can obtain one at http://mozilla.org/MPL/2.0/.
+* SPDX-License-Identifier: MPL-2.0
+*
+* This file is part of Dynawo, an hybrid C++/Modelica open source time domain simulation tool for power systems.
+*/
+
+model LoadAlphaBetaRestorative "Generic model of a restorative alpha-beta load."
+  extends BaseClasses.BaseLoad;
+  extends AdditionalIcons.Load;
+
+  public
+    parameter Types.Time tFilter "Time constant of the load restoration";
+    parameter Types.VoltageModulePu UMinPu "Minimum value of the voltage amplitude at terminal in p.u (base UNom) that ensures the P/Q restoration";
+    parameter Types.VoltageModulePu UMaxPu "Maximum value of the voltage amplitude at terminal in p.u (base UNom) that ensures the P/Q restoration";
+    parameter Real alpha = 2 "Active load sensitivity to voltage";
+    parameter Real beta = 2 "Reactive load sensitivity to voltage";
```

Comment:
Don't use a default value here and add a capital letter (coding style)

Reply:
Done